sdk/keyvault/keyvault-keys/package.json

```diff
@@ -72,6 +72,7 @@
     "@azure/core-http": "^1.0.0-preview.1",
     "@azure/core-paging": "^1.0.0-preview.1",
     "@azure/identity": "^1.0.0-preview.1",
+    "@types/query-string": "6.2.0",
```

Comment:
Why is `@types/query-string` a full dependency while `query-string` is only a dev dependency?

Reply:
it's a mistake! thanks for pointing it out!